docs/fedora-releases-guide.md

```diff
@@ -18,6 +18,83 @@ Doing Fedora releases with Packit means utilising these jobs:
 
 Every job takes care of a different part of the release process.
 
+## Propose downstream and pull from upstream comparison
```

Comment:
Could you please add a sentence about what these two jobs try to achieve in general before the tables? I am worried it might look very technical from the very start..;)

Something like this:

```suggestion
## Propose downstream or pull from upstream

There are two jobs that can help you to get your new release to Fedora.
They differ in the way they are triggered and configured but share the implementation.

The push workflow is configured and started in the upstream repository,
unlike the pull workflow that is configured in dist-git.

Here are the key differences between the two:
```